### PR TITLE
Update UGDG for omegafishy PPP

### PR DESCRIPTION
**Pull request by omegafishy**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -448,11 +448,13 @@ image::AddmhViewmhSequenceDiagram.png[width="800"]
 
 * Command execution ends.
 
-
 ****
 NOTE: If multiple patients with the entered `NRIC` exist, then the `AddmhCommand::execute` will throw a `CommandException`
 with an appropriate error message before the use case ends.
 ****
+
+==== Design Considerations
```
Comment (snajef) [inline docs/DeveloperGuide.adoc]: Is there any intention to add anything more here? Seems odd to have an empty region here.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -101,19 +101,34 @@ Example(s):
...
-* If the NRIC does not match to an existing patient in the system, an error will be displayed.
-* If the format of the doctor's name is incorrect, an error will be displayed.
+WARNING: The following invalid inputs will result in a command failure, and the display of an appropriate error message.
+
+* Invalid NRIC
+** The patient NRIC does not match to an existing patient in the system. The person will first need to be registered.
+* Incorrect format of the doctor's name.
+** Doctor's title must be included.
+** The first letter of all words in doctor's name must be captalised.
+* Blank Diagnosis
```
Comment (snajef) [inline docs/UserGuide.adoc]: Just wondering if Diagnosis should or should not be capitalised here

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -270,7 +285,7 @@ See <<#view-command, entry>> on the `view` command.
 
 Example(s):
 
-* `viewmh ic/S1234567A`
+* `view mh ic/S1234567A`
```
Comment (snajef) [inline docs/UserGuide.adoc]: is the ic/ tag here intentional?

File: docs/team/omegafishy.adoc [hunk 4]
```diff
@@ -37,12 +41,15 @@ These features include (among others):
 * *Major enhancement*: Added the functionality to record and view patients' past medical diagnoses
 ** What it does: Records a diagnosis entry into the medical history of a given patient. Each entry includes the diagnosis description, the name of the examining and a timestamp.
 ** Justification: This feature serves as the primary storage of important patient data, as adjudged by the examining doctor's assessment. Past diagnoses are easily viewed on the desktop application to provide quick access for the examining doctor.
-//** Highlights: The feature is
-//** Credits:
+** Highlights: This enhancement required a detailed understanding of the Java Architecture for XML Binding software frame work (JAXB) for data storage as XML files.
+Also, the integration of the UI in the feature needed JAVAFX know-how.
```
Comment (snajef) [inline docs/team/omegafishy.adoc]: Minor nitpick, should JAVAFX be JavaFX?

File: docs/UserGuide.adoc (hunk 3 above)
Comment (omegafishy) [reply docs/UserGuide.adoc]: Good call!

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (omegafishy) [reply docs/DeveloperGuide.adoc]: It has been filled, will update it soon!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (omegafishy) [reply docs/UserGuide.adoc]: It shouldn't be capitalised, thanks for pointing out.

File: docs/team/omegafishy.adoc (hunk 4 above)
Comment (omegafishy) [reply docs/team/omegafishy.adoc]: Thanks for spotting this typo.

**Pull request merged**